boards/shields/ls0xx_generic/Kconfig.defconfig

```diff
@@ -13,10 +13,6 @@ config LS0XX
 
 if LVGL
 
-config LVGL_DISPLAY_DEV_NAME
-	string "Display device name"
-	default "LS0XX"
-
 config LVGL_HOR_RES_MAX
 	int "Horizantal resolution"
 	default 128
```

Comment:
Do you also need a separate patch removing these? These values should be coming from devicetree now, right?

Reply:
Yes, except for mimxrt10xx boards.